<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Translator</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLang1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLang2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtLang1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>281</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtLang2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>281</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTranslate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Translate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
